pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "xocto"
-version = "5.1.0"
+version = "5.1.1"
```

Comment:
should this be 6.0.0?

Reply:
Sorry yes, made that change before I noticed the breaking change 🤦

Reply:
Fixed